httpdocs/js/user.js

```diff
@@ -280,12 +281,13 @@ export default class User extends Router {
   }
   updateDisplayName() {
     let that = this;
+    const emailBeginning = that.email.substring(0, that.email.indexOf("@"));
```

Comment:
Shouldn't we move this declaration inside the if scope below?

Reply:
Yes, this would be better.